core-plugin/testSrc/com/google/gct/idea/debugger/CloudExecutionStackTest.java

```diff
@@ -0,0 +1,103 @@
+package com.google.gct.idea.debugger;
+
+import com.google.api.services.clouddebugger.model.SourceLocation;
+import com.google.api.services.clouddebugger.model.StackFrame;
+import com.google.api.services.clouddebugger.model.Variable;
+
+import com.intellij.openapi.project.Project;
+import com.intellij.ui.SimpleTextAttributes;
+import com.intellij.xdebugger.frame.XCompositeNode;
+import com.intellij.xdebugger.frame.XDebuggerTreeNodeHyperlink;
+import com.intellij.xdebugger.frame.XValueChildrenList;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.swing.Icon;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CloudExecutionStackTest {
+
+  @Mock
+  private Project project;
+
+  @Test
+  public void testGetTopFrame_null() {
```

Comment:
testGetTopFrame_null() doesn't make it very clear what is supposed to be null or why.